### DISCUSSION TITLE
Add support for pasting parameters in Regex triggers

### DISCUSSION THREAD

**Discussion by Sun0fBeach** [espanso/espanso#1266] [Feature requests and Ideas]
**Describe the bug**
I use a regex trigger to put the parameter into my replace
until then no problemo, but the parameter that i want to insert is quite long (but still shorter than the limite of 30 characters) so I use a copy and past (CTRL+V) to put my parameter in my regex trigger. And there is the mater :
it start my command :regex(XXXXXXXXXX) but when I hit space the replace look like :

:regex(XXthen the rest of the replace with my broken {{parameter}} as a square symbol

**To Reproduce**
Simply CTRL+V the parameter into the regex trigger

**Expected behavior**
Do my replace as it do when I type it all the way

**Screenshots**
![regex1](https://user-images.githubusercontent.com/25881253/160259370-de11fac3-dc8b-4d60-be1c-8e7c4c35a1ad.jpg)
![regex2](https://user-images.githubusercontent.com/25881253/160259191-2b1c7c5f-0ce8-44ca-9a04-9fb8577c3c46.jpg)

**Setup information**
- OS: Win
- Version: 2.1.4-beta

**Comment by federico-terzi** [comment]
Hey @Sun0fBeach,

I'm sorry you experienced this problem! Just to make sure I understand, are you trying to insert the regex parameter by pasting it inside the parenthesis?

Cheers :)

**Comment by Sun0fBeach** [comment]
@federico-terzi yes you understood well ;)

**Comment by federico-terzi** [comment]
@Sun0fBeach I see, thanks! Currently, that's not supported (though we'll try to add support in the future). For now, you'll need to manually type the parameter or use [Forms](https://espanso.org/docs/matches/forms/)

Cheers :)
